### PR TITLE
Finnish translation v.1

### PR DESCRIPTION
**Pull request by istenrot**
Pull request for a Finnish translation.
Closing issue #34 

Proof reading still ongoing. Updates for the PR may come.

### CODE REVIEW

**Comment by istenrot** [comment]
First proof reading done.

**Comment by ahunts** [comment]
Would you mind taking a look at the other Finnish translation? I have put in there two other Finnish translations but we are trying to get the best out of everyone's! Thanks

**Comment by ViliLipo** [comment]
@istenrot  sorry I missed your issue, when I begun on working my translation. Should we sync the Finnish translation process to one PR now and close the other one?

**Comment by ahunts** [comment]
@ViliLipo yes that would be great! I think they are almost done but the more people who work on it the better I think!

**Comment by istenrot** [comment]
@ahunts @ViliLipo do you have open PRs I could take a look at? Maybe link all versions to issue #34 and continue discussion over there.

**Comment by istenrot** [comment]
@ahunts @ViliLipo I did merge your contributions to mine. Let's try to get the first version out.

**Comment by istenrot** [comment]
@flore2003 This is ready now for the mege.

**Comment by istenrot** [comment]
Anyone knowing Finnish, please review this PR so this can be merged.

File: site/globals/langs.json [hunk 1]
```diff
@@ -83,5 +83,10 @@
         "code": "ja",
         "url": "/ja/",
         "name": "日本語"
+    },
+    {
+        "code": "fi",
+        "url": "/fi/",
+        "name": "Suomi"
```
Comment (kekkis) [inline site/globals/langs.json]:
```suggestion
        "name": "suomi"
```

File: site/fi.njk [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+                <li class="pb-2">Pese kätesi usein. Yski ja aivasta oikeaoppisesti.</li>
+                <li class="pb-2">Yritä olla koskematta kasvoja, etenkin suuta, nenää ja silmiä.</li>
+                <li class="pb-2">Pidä etäisyyttä muihin ihmisiin, ei haleja, suudelmia, kättelyitä tai yläfemmoja. Käytä tarvittaessa turvallisempia vaihtoehtoja.</li>
+                <li class="pb-2">Älä käy konserteissa, esityksissä, urheilutapahtumissa tai muissa suurissa viihdetapahtumissa.</li>
+                <li class="pb-2">Pidättäydy museokäynneistä, näyttelyvierailuista, elokuvateattereissa, yökerhoissa ja muissa kulttuurikohteissa käymisestä.</li>
+                <li class="pb-2">Pysy poissa kokoontumisista ja tapahtumista, kuten kerhoista, uskonnollisista toimituksista ja yksityisistä juhlista.</li>
+                <li class="pb-2">Matkusta niin vähän kuin mahdollista. Älä tee kaukomatkoja, ellei se ole välttämätöntä.</li>
+                <li class="pb-2">Älä käytä julkista liikennettä, ellei se ole välttämätöntä.</li>
+                <li class="pb-2">Tee töitä kotoa käsin, jos se on mahdollista. Suostuttele työnantajaasi sallimaan etätyöskentely tarvittaessa.</li>
```
Comment (mirasusanna) [inline site/fi.njk]: työnantajaasi -> työnantajasi

File: site/fi.njk [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Eksponentiaalinen kasvu</h3>
+                    <p class="font-normal">
+                        Immuniteetin puutteen johdosta, ihmispopulaation koko on yläraja mahdollisten tartuntojen määrälle. Vaikka alkuun eksponentiaalinen
+                        kasvu vaikuttaa aina hitaalta, se voi johtaa käsittämättömän suuriin lukumääriin hyvin lyhyellä aikavälillä.
+                        Kun sairastuneiden lukumäärä kaksinkertaistuu muutamassa päivässä
+                        <sup><a class="source-link" href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports/" target="_blank">[7]</a></sup>
+                        terveydenhuoltojärjestelmämme voivat mahdollisesti ylikuormittua, joka johtaa isoon määrään kuolemantapauksia, koska ihmiset eivät saa
```
Comment (mirasusanna) [inline site/fi.njk]: terveydenhuoltojärjestelmämme voivat ylikuormittua, mikä...

File: site/fi.njk [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+                <iframe class="ml-6" src="https://www.facebook.com/plugins/share_button.php?href=https%3A%2F%2Fstaythefuckhome.com&layout=button_count&size=large&appId=764658637392408&width=111&height=28" width="111" height="28" style="border:none;overflow:hidden" scrolling="no" frameborder="0" allowTransparency="true" allow="encrypted-media"></iframe>
+            </div>
+
+            <p class="text-sm text-gray-500 mb-10 mt-10">
+                Kaikille jotka haluavat osallistua #StayTheFuckHome-sivuston ja liikkeen kehittämiseen, tiedoksi, että olemme tehneet lähdekoodin avoimeksi
+                kehitykseen osallistumisen mahdollistamiseksi. GitHub-koodilähde sijaitsee osoitteessa: <a href="https://github.com/flore2003/staythefuckhome" target="_blank">https://github.com/flore2003/staythefuckhome</a>
+                <br />
+                Jos sinulla on mitä tahansa muuta palautetta liittyen sivuston sisältöön, voitte kirjoittaa meille osoitteeseen
```
Comment (mirasusanna) [inline site/fi.njk]: voitte -> voit

File: site/fi.njk [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+                <br /><br />
+                Sanotaan se nyt suoraan: <strong>Pysy Hitto Kotona!</strong>
+            </p>
+
+            <h2 class="text-3xl">Itsekaranteenin manifesti</h2>
+
+            <p>
+                Ilman tutkittuja hoitoja ja käyttökelpoista rokotetta, jonka kehittämiseen kuluu vielä ainakin vuosi, ainoa tehokas keino
+                estää koronaviruspandemian räjähtämistä käsiin on antaa sille vähemmän mahdollisuuksia levitä. Seuraavan listan toimenpiteet,
+                järjestettynä helpommasta tehokkaimpaan tapaan torjua pandemiaa, ovat ohjenuoria heille, jotka haluavat liittyä liikkeeseen
```
Comment (mirasusanna) [inline site/fi.njk]: helpommasta -> helpoimmasta?

File: site/fi.njk [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+            <p class="text-xl">
+                Muistathan, että ei ole olemassa oikeaa tai väärää määrää toimenpiteitä. Ota käyttöön toimenpiteet, jotka sopivat sinulle ja jotka eivät
+                vaaranna elinympäristöäsi tai toimeentuloasi. <strong>Älä lopeta töitäsi tämän vuoksi!</strong>
+                Kuitenkin muista, että jokainen toimenpide on avuksi.
+            </p>
+
+            <h2 class="text-3xl">Miksi tämä on niin tärkeää</h2>
+
+            <p>
+                SARS-CoV-2 on helposti tarttuva ja joissain tapauksissa tappava virus, joka aiheuttaa hengityselinsaurauuta nimeltään COVID-19.
```
Comment (mirasusanna) [inline site/fi.njk]: hengityselinsaurauuta -> hengityselinsairautta

File: site/fi.njk (hunk 5 above)
Comment (mirasusanna) [inline site/fi.njk]: helpommasta -> helpoimmasta?

File: site/fi.njk (hunk 6 above)
Comment (mirasusanna) [inline site/fi.njk]: hengityselinsaurauuta -> hengityselinsairautta

**Review by mirasusanna — APPROVED** [review]
Overall a fine translation, I left a couple improvement suggestions and found one typo.

**Review by mirasusanna — APPROVED** [review]

File: site/fi.njk [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+                        ja rokotteen kehittäminen vaatii huomattavasti aikaa.
+                        Ilman rokotetta ja aiempaa kattavaa immuniteettia kaikki ovat alttiita tartunnalle.
+                        Vaikka suurin osa tartunnan saaneista henkilöistä saavat vain lieviä oireita, laumasuojan puuttuminen voi johtaa riskiryhmiin kuuluvien
+                        sairastumisiin merkittävissä määrin.
+                    </p>
+                </li>
+                <li class="pb-2">
+                    <h3>Eksponentiaalinen kasvu</h3>
+                    <p class="font-normal">
+                        Immuniteetin puutteen johdosta, ihmispopulaation koko on yläraja mahdollisten tartuntojen määrälle. Vaikka alkuun eksponentiaalinen
```
Comment (hirvinen) [inline site/fi.njk]: A very slight change in semantics, but much more fluent Finnish would be e.g.
```suggestion
                        Immuniteetin puutteen johdosta yläraja mahdollisten tartuntojen määrälle on koko ihmiskunta. Vaikka alkuun eksponentiaalinen
```

**Review by hirvinen — APPROVED** [review]

File: site/globals/langs.json (hunk 1 above)
Comment (kekkis) [reply site/globals/langs.json]: Very minor but the language name in Finnish is 'suomi'.

**Review by kekkis — APPROVED** [review]

File: site/fi.njk (hunk 2 above)
Comment (istenrot) [reply site/fi.njk]: Thanks!

**Comment by tirri** [comment]
Hi! I'm quite new to using this so it appears that my edits went here: https://github.com/flore2003/staythefuckhome/pull/113. What do you think would be the best way to proceed? Awfully sorry if there's extra work for you. :/

**Comment by phelmig** [comment]
Thank you all for your efforts. Please comment here with your name if I should add you to the list of contributors:
https://github.com/flore2003/staythefuckhome/pull/110

**Comment by phelmig** [comment]
Thank you all for your efforts. Please comment here with your name if I should add you to the list of contributors:
https://github.com/flore2003/staythefuckhome/pull/110

**Comment by phelmig** [comment]
Can everyone confirm that https://github.com/flore2003/staythefuckhome/pull/113 is fine as a final version? Also please let me know if I should add you to the contributors file!

**Comment by istenrot** [comment]
@phelmig I agree that #113 is the (current) final version. I can be added to the list of contributors.

**Pull request closed without merging**